ncdiff/src/yang/ncdiff/runningconfig.py

```diff
@@ -97,6 +97,7 @@
     (re.compile(r'^ *mpls mldp static '), 0),
     (re.compile(r'^ *router '), 0),
     (re.compile(r'^ *summary-address '), 1),
+    (re.compile(r'^ *distribute-list 1300 out '), 2),
```

Comment:
Would a more general one work better? 1300 or other access-list, prefix or route-map are possible.
`(re.compile(r'^ *distribute-list '), 2),`

Reply:
I have made changes as you suggested.please check check it

Passed log:
https://earms-trade.cisco.com/tradeui/logs/details?archive=/nobackup/vijviswa/cmt_script/polaris/binos/linkfarm/sdks-host/iosxe-pyats/nativesdk/sdk/sysroots/x86_64-xesdk-linux/usr/users/vijviswa/archive/24-06/cli2show_job.2024Jun19_12:13:16.655979.zip&atstype=ATS